subsys/bluetooth/mesh/shell/shell_utils.c

```diff
@@ -0,0 +1,182 @@
+/*
```

Comment:
You can remove this (and corresponding header) file from PR after generic PR gets merged and only retain changes.